packages/block-library/src/spacer/edit.js

```diff
@@ -52,17 +74,16 @@ const SpacerEdit = ( {
 					bottomLeft: false,
 					topLeft: false,
 				} }
-				onResizeStart={ onResizeStart }
-				onResizeStop={ ( event, direction, elt, delta ) => {
-					onResizeStop();
-					const spacerHeight = Math.min(
-						parseInt( height + delta.height, 10 ),
-						MAX_SPACER_HEIGHT
-					);
-					updateHeight( spacerHeight );
-				} }
+				onResizeStart={ handleOnResizeStart }
+				onResizeStop={ handleOnResizeStop }
 				showHandle={ isSelected }
-			/>
+			>
+				<ResizeTooltip
+					axis="y"
+					isVisible={ isResizing }
+					position="bottom"
+				/>
```

Comment:
Have you considered rendering actually _in_ ResizableBox? If there are reasons to keep it separate, do let me know. Otherwise, it might be a boon in developer experience that ResizableBox could handle everything related to resizing on its own.

I can't help but think that `axis` and `position` can be derived from ResizableBox's `enable` and to some extent `size` props. If we were to move Tooltip into Box the consumer of the component wouldn't have to worry about such extra props.

Reply:
@mcsf I have considered putting it in `ResizableBox` :). That's how this idea started out.
The reason I thought of keeping it independent was for added flexibility. That is, to show dimensions on resize that aren't triggered by `ResizableBox`. For example, browser `window` resize.

To start, we could have this be a default (experimental) feature of `ResizableBox`. Something like `<ResizableBox __experimentalShowTooltip={true} />` would show the Tooltip.

What do you think? :)

Reply:
>  The reason I thought of keeping it independent was for added flexibility. That is, to show dimensions on resize that aren't triggered by `ResizableBox`. For example, browser `window` resize.

Hm, I could see that as an interaction, although personally I would wonder _when_ we actually expect users to resize their proper window in a context where we'd show the tool tip. :)

So I think we're still in this territory: "the position I tend to come from is to minimise how much is exposed at first, and see over time what should be opened up". Internally, ResizeTooltip is a good component to build separately, but I would **keep it private** to our components.

> To start, we could have this be a default (experimental) feature of `ResizableBox`. Something like `<ResizableBox __experimentalShowTooltip={true} />` would show the Tooltip.

I'd be happy to see that experiment. :) Don't feel like you need to do it in this PR, though, as long as we keep the right things private/experimental.

Reply:
> Internally, ResizeTooltip is a good component to build separately, but I would keep it private to our components.

@mcsf Sounds good to me 👍 . With this PR, in it's current state, does exporting it as `__experimental` qualify? Or would it be better to integrate it into `ResizableBox` and not export it at all?

Thanks!

Reply:
> in it's current state, does exporting it as `__experimental` qualify? Or would it be better to integrate it into `ResizableBox` and not export it at all?

If you don't mind too much, integrating in `ResizableBox` and not exporting would be the best, IMO!

Reply:
@mcsf Alrighty! Let's go with that for now :)